hivemq-edge/src/frontend/src/modules/Workspace/components/nodes/NodeAdapter.tsx

```diff
@@ -39,13 +41,13 @@ const NodeAdapter: FC<NodeProps<Adapter>> = ({ id, data: adapter, selected }) =>
             <IconButton
               icon={<Icon as={deviceCapabilityIcon['WRITE']} />}
               aria-label={t('workspace.toolbar.command.subscriptions.outward')}
-              // onClick={onCreateGroup}
+              onClick={() => navigate(`/workspace/node/adapter/${adapter.type}/${id}/outward`)}
             />
           )}
           <IconButton
             icon={<Icon as={deviceCapabilityIcon['READ']} />}
             aria-label={t('workspace.toolbar.command.subscriptions.inward')}
-            // onClick={onCreateGroup}
+            onClick={() => navigate(`/workspace/node/adapter/${adapter.type}/${id}/inward`)}
```

Comment:
suggestion (not important): I would have defined a variable for the common part of the navigation path
e.g.
```ts
const adapterNavPath = `/workspace/node/adapter/${adapter.type}/${id}`
          <IconButton
              icon={<Icon as={deviceCapabilityIcon['WRITE']} />}
              aria-label={t('workspace.toolbar.command.subscriptions.outward')}
              onClick={() => navigate(`${adapterNavPath}/outward`)}
            />
          <IconButton
            icon={<Icon as={deviceCapabilityIcon['READ']} />}
            aria-label={t('workspace.toolbar.command.subscriptions.inward')}
            onClick={() => navigate(`${adapterNavPath}/inward`)}
          />

```